Packaging/OpenDingux/control

```diff
@@ -0,0 +1,10 @@
+Package: devilutionX
+Version: 0.0.1
+Architecture: mipsel
```

Comment:
Very exciting to see that it is a MIPS platform, we have been wanting to see the game run on this since the original was also ported to PSX (MIPS R3000).